collect_app/src/main/java/org/odk/collect/android/database/instances/InstanceDatabaseMigrator.java

```diff
@@ -205,4 +214,25 @@ public void createInstancesTableV8(SQLiteDatabase db) {
                 + GEOMETRY + " text, "
                 + GEOMETRY_TYPE + " text);");
     }
+
+    public void createInstancesTableV9(SQLiteDatabase db) {
+        db.execSQL("CREATE TABLE IF NOT EXISTS " + INSTANCES_TABLE_NAME + " ("
+                + _ID + " integer primary key autoincrement, "
+                + DISPLAY_NAME + " text not null, "
+                + SUBMISSION_URI + " text, "
+                + CAN_EDIT_WHEN_COMPLETE + " text, "
+                + CAN_DELETE_BEFORE_SEND + " text, "
+                + INSTANCE_FILE_PATH + " text not null, "
+                + JR_FORM_ID + " text not null, "
+                + JR_VERSION + " text, "
+                + STATUS + " text not null, "
+                + LAST_STATUS_CHANGE_DATE + " date not null, "
+                + DELETED_DATE + " date, "
+                + GEOMETRY + " text, "
+                + GEOMETRY_TYPE + " text, "
+                + EDIT_OF + " integer REFERENCES " + INSTANCES_TABLE_NAME + "(" + _ID + ") CHECK (" + EDIT_OF + " != " + _ID + "),"
```

Comment:
I think we need to give up on this because it's problematic. When we edit forms and then delete the original one, it will crash since other forms will be pointing to the non-existent one. The fact that we delete these forms and effectively free up their IDs is fine because the database definition says that the same IDs cannot be reused, so we're safe and don't need to do anything. ANy thoughts @seadowg